PySDM/physics/isotope_kinetic_fractionation_factors/bolot_et_al_2013.py

```diff
@@ -0,0 +1,54 @@
+"""
+kinetic fractionation factor from [Bolot et. al. 2013](https://doi.org/10.5194/acp-13-7903-2013)
+eq. (1) with subsequent variable definitions on pages 7906 and 7907 and in Appendix (A6, A6).
+In the paper, alpha_kinetic is probably actually alpha_effective
+(contains alpha_equilibrium in numerator).
+q_sat_inf in Bolot eq (A7) is substituted with equivalent form of (1/S * rho_v_inf/rho_[l,i]_inf).
+"""
+
+
+class BolotEtAl2013:  # pylint: disable=too-few-public-methods
+    def __init__(self, _):
+        pass
+
+    @staticmethod
+    def alpha_kinetic(
+        alpha_equilibrium,
+        relative_humidity,
+        heavy_to_light_diffusivity_ratio,
+        heavy_to_light_ventilation_ratio,
+        lv,
+        cp,
+        fv,
+        fh,
+        Le,
+        R_v,
+        T_inf,
+        water_vapor_density,
+        condensed_water_density,
+    ):  # pylint: disable=too-many-positional-arguments too-many-arguments
+        """
+        water_vapor_density - density of water vapor at ambient temperature
+        condensed_water_density - density of liquid or ice at ambient temperature
+        """
+        Ai = 1 / (
```

Comment:
Let's aim at reusing the code for the thermal inertia factor by linking to the Howell/Mason "factors":
(see also https://github.com/open-atmos/PySDM/issues/1554)
https://github.com/open-atmos/PySDM/blob/1b0013d62bd8d37593331af82a061990a5660c31/PySDM/physics/drop_growth/mason_1971.py#L18-L19

Passing a `howell_factor` argument would also avoid recomputing the same factor for millions of super-droplets - the only dependence on particle properties (size) seem to come from the ventilation factor?

Reply:
this is also relevant: https://github.com/open-atmos/PySDM/blob/1b0013d62bd8d37593331af82a061990a5660c31/PySDM/backends/impl_numba/methods/deposition_methods.py#L89-L101